Created a README and a function for always run (#4622)
openshift/ci-tools: cmd/ci-operator-prowgen/README.md

prowgen
=======

Prowgen is a tool that converts [ci-operator configuration](https://docs.ci.openshift.org/docs/architecture/ci-operator/)

> into [Prowjobs](https://docs.prow.k8s.io/docs/jobs/).

Reviewer: Prowgen doesn't generate prowjobs. https://github.com/openshift/ci-tools/pull/4622/files#r2189669641

> The objetive is to made `ci-operator configuration` the only source of truth for jobs created

Reviewer 2: ```suggestion
The objective is to make `ci-operator configuration` the only source of truth for jobs created
```

by **prowgen**, based on configurations inside `openshift/ci-operator/config/`, replacing any

> user manual modification directly on `openshift/ci-operator/jobs/`.

Reviewer 2: ```suggestion
user modifications directly on `openshift/ci-operator/jobs/`.
```

prowgen
=======

Prowgen is a tool that generates [job configurations](https://docs.prow.k8s.io/docs/jobs/) based on
[ci-operator configuration](https://docs.ci.openshift.org/docs/architecture/ci-operator/) and its own
configuration file named `.config.prowgen`.

The contents of `.config.prowgen` will be appended to every job configuration during Prowgen execution:

> **Exemple:**

Reviewer: ```suggestion
**Example:**
```
